vcluster/deploy/topologies/air-gapped.mdx

```diff
@@ -147,110 +100,143 @@ chmod +x ./push-images.sh
+    
+    :::tip
+    You can remove images from `images-optional.txt` that you don't need as multiple Kubernetes versions and all available Kubernetes distributions are in the file.
+    :::
+
+    ```bash title "Pull and push all optional images"
+    wget https://github.com/loft-sh/vcluster/releases/download/v${VCLUSTER_VERSION}/images-optional.txt
+    ./download-images.sh --image-list images-optional.txt
+    ./push-images.sh --registry ${REGISTRY}
+    ```
 
+    
+  </Step>
 </Flow>
-## Configure vCluster for air-gapped install
 
-The `vcluster.yaml` file holds your vCluster configuration and allows overriding the vCluster Helm chart default values.
+## Populate vCluster Helm Chart to private registry
 
-Edit your existing `vcluster.yaml` file or create a new one with the following content:
+### Pre-requisites 
 
-:::tip
-To retrieve supported Kubernetes versions:
+These need to be available on a machine that has internet access that will push and pull the helm chart to the private registry:
 
-```bash title="Show supported Kubernetes versions"
-curl http://"${REGISTRY}"/v2/kube-controller-manager/tags/list
-```
+- Ability to push to your OCI-compatible private image regsitry
+- `curl` installed
+- `helm` installed: Helm v3.10+
 
-```bash title="create vcluster.yaml configuration"
-export KUBERNETES_VERSION="v1.31.1"  # Replace with the desired version
-cat <<EOF > vcluster.yaml
-controlPlane:
- advanced:
-   defaultImageRegistry: ${REGISTRY}
- distro:
-   k8s:
-     version: ${KUBERNETES_VERSION}
-EOF
-```
-
-:::info Example of a fully formed registry path
-If your `REGISTRY` is set to `ecr.io/myteam`, a fully formed registry path might look like:
-`ecr.io/myteam/ghcr.io/loft-sh/vcluster:0.21.0`
-:::
 
+### Push vCluster Helm chart
 
-:::info Note on alpine image replacement
-The `alpine` image may be replaced with any similar image with the following `vcluster.yaml` configuration:
+<Flow id="offline-helm-chart">
+<Step>
 
-```yaml title="vcluster.yaml configuration for custom image"
-sync:
-  toHost:
-    pods:
-      rewriteHosts:
-        initContainer:
-          image: your-registry/your-image:1.0.0
+```bash title="Download the vCluster Helm chart"
+export VCLUSTER_VERSION="0.25.0"  # Replace with the desired version
+curl -O https://charts.loft.sh/charts/vcluster-"${VCLUSTER_VERSION}".tgz
 ```
+</Step>
 
-The image that is used to replace the default image must be able to run the following command:
+<Step>
+
+```bash title="Push your chart to your private registry"
+export REGISTRY=ecr.io/myteam # This should be a prefix; do not include any image paths
 
-```yaml
-Command: []string{"sh"},
-Args:    []string{"-c", "sed -E -e 's/^(\\d+.\\d+.\\d+.\\d+\\s+)" + fromHost + "$/\\1 " + toHostnameFQDN + " " + toHostname + "/' /etc/hosts > /hosts/hosts"}
+helm chart save ./vcluster-${VCLUSTER_VERSION}.tgz ${REGISTRY}/vcluster:v${VCLUSTER_VERSION}
```

Comment:
These worked for me instead:

```
helm registry login my-private-registry:5000
helm push vcluster-"${VCLUSTER_VERSION}".tgz oci://my-private-registry:5000/vcluster
```
And pulling this
```
helm pull oci://my-private-registry:5000/vcluster/vcluster:${VCLUSTER_VERSION}
tar -xvf vcluster-"${VCLUSTER_VERSION}".tgz
```

Reply:
Those instructions are incorrect. I don't need you to install the chart locally, but need you to try to install using directly from the oci registry.